PlayerController.cs

```diff
@@ -18,19 +23,19 @@ void Update()
         
     }
 
-    [Header("General Setup Settings")]
-    [SerializeField] private InputAction movement;
-    [Tooltip("How fast player moves up and down based upon player input")][SerializeField] float controlSpeed = 30f;
-    [Tooltip("How far player moves horizontally")][SerializeField] float xRange = 10f;
-    [Tooltip("How far player moves vertically")][SerializeField] float yRange = 10f;
+    [Header("General Setup Settings")] // UI header title
```

Comment:
These are called [Unity Attributes](https://docs.unity3d.com/Manual/Attributes.html). In a way, it is technically UI where it is a new line of interface for Unity's Inspector Editor. Usually, we refer to UI for Unity's Inspector to be used with UIToolkit and UI for games would be used with UIToolkit and uGUI.